Sources/AuthFoundation/Responses/OAuth2ServerError.swift

```diff
@@ -33,3 +33,35 @@ public struct OAuth2ServerError: Decodable, Error, LocalizedError {
         case description = "errorDescription"
     }
 }
+
+extension OAuth2ServerError {
+    ///  Possible  OAuth 2.0 server error code
+    public enum OAuth2ServerErrorCode: String, Decodable {
+        /// The authorization request is still pending as the end user hasn't yet completed the user-interaction step
+        case authorization_pending
```

Comment:
Can you camelCase these enums, to be more "swifty"?

Reply:
Definitely. I've made the change.